src/mpyl/schema/project.schema.yml

```diff
@@ -773,6 +773,15 @@ definitions:
         additionalProperties: false
       portMappings:
         type: object
+      imagePullSecrets:
+        minItems: 1
+        description: 'ImagePullSecrets is a list of references to secrets in the same namespace
+          to use for pulling any images in pods that reference this ServiceAccount. ImagePullSecrets
+          are distinct from Secrets because Secrets can be mounted in the pod, but ImagePullSecrets
+          are only accessed by the kubelet. More info: https://kubernetes.io/docs/concepts/containers/images/#specifying-imagepullsecrets-on-a-pod'
+        items:
+          "$ref": k8s_api_core.schema.yml#/definitions/io.k8s.api.core.v1.LocalObjectReference
+        type: [ array, null ]
```

Comment:
Shouldn't this be part of the mpyl_config schema?

Reply:
You can configure this per project and in `mpyl_config.yml` you can set the defaults. That's why `mpyl_config.yml` embeds parts of the `project.yml` schema.

Reply:
Yes but I just tried it and the mpyl config schema validation fails because of the unexpected imagePullSecrets field

Reply:
Where did the schema validation fial? The unittests validate the schema here https://github.com/Vandebron/mpyl/blob/2e179fcc4db22dac3b2563a2804bcbfaa68c1a0e/tests/test_resources/mpyl_config.yml#L90-L92

Reply:
I added the image pull secrets to the mpyl config, added a dummy change to a project then used this test version and the build failed with 

```
[14:28:50] Schema validation failed Additional properties are not allowed ('imagePullSecrets' was unexpected) at `kubernetes`                                                                           
           Unexpected exception: Additional properties are not allowed ('imagePullSecrets' was unexpected)   
```

it fails at https://github.com/Vandebron/mpyl/blob/b450a7030cb14c4465bae517e034690e628c6584/src/mpyl/build.py#L107-L118 when constructing the `Steps` object (when the mpyl_config.yml validation happens)

Reply:
ah maybe I used the wrong place to put the config

Reply:
yep wrong kubernetes section :/